src/clusterfuzz/_internal/bot/fuzzers/syzkaller/config.py

```diff
@@ -64,6 +66,14 @@ def generate(serial,
     data['target'] = 'linux/amd64'
     data['disable_syscalls'] = ['openat$vhost_vsock']
     data['sandbox'] = 'none'
+    device = {}
+    device['serial'] = serial
+    # Syzkaller uses cuttlefish kernel.log as console instead of serial console.
+    # kernel.log will be collected within respective cuttlefish_runtime folder.
+    # We only have one instance launched at one time.
+    cvd_dir = environment.get_value('CVD_DIR')
+    device['console'] = f'{cvd_dir}/cuttlefish_runtime/kernel.log'
```

Comment:
if this file is not found, does it throw a warning log and keep running? or does it error and crash? just asking for fuzzit